projects/examples/src/components/quick-search/filters/quick-search-filters.example.component.html

```diff
@@ -0,0 +1,22 @@
+Press <kbd>Command</kbd>+<kbd>f</kbd> on MacOS / <kbd>Ctrl</kbd>+<kbd>f</kbd> on PC or click
```

Comment:
I don't see anything here referring to explaining to use the filter icon and that it will filter for results of that service provider type. 

Is this copy pasted from another example? Why is there an input with a placeholder?

Please rework thinking about a user that has never seen the feature before and is coming to this specific examples

Reply:
Make sure we explain both ways, using the mouse, or just the keyboard. We want to explain the feature in these examples